spec/dnsimple/client/domains_spec.rb

```diff
@@ -133,28 +133,29 @@
     end
 
     it "builds the correct request" do
-      subject.domain(account_id, domain = "example.com")
+      subject.domain(account_id, domain = "example-alpha.com")
 
       expect(WebMock).to have_requested(:get, "https://api.dnsimple.test/v2/#{account_id}/domains/#{domain}")
           .with(headers: { 'Accept' => 'application/json' })
     end
 
     it "returns the domain" do
-      response = subject.domain(account_id, "example.com")
+      response = subject.domain(account_id, "example-alpha.com")
       expect(response).to be_a(Dnsimple::Response)
 
       result = response.data
       expect(result).to be_a(Dnsimple::Struct::Domain)
-      expect(result.id).to eq(1)
-      expect(result.account_id).to eq(1010)
-      expect(result.registrant_id).to eq(nil)
+      expect(result.id).to eq(181984)
+      expect(result.account_id).to eq(1385)
+      expect(result.registrant_id).to eq(2715)
       expect(result.name).to eq("example-alpha.com")
-      expect(result.state).to eq("hosted")
+      expect(result.state).to eq("registered")
       expect(result.auto_renew).to be(false)
       expect(result.private_whois).to be(false)
-      expect(result.expires_on).to eq(nil)
-      expect(result.created_at).to eq("2014-12-06T15:56:55Z")
-      expect(result.updated_at).to eq("2015-12-09T00:20:56Z")
+      expect(result.expires_at).to eq("2021-06-05T02:15:00Z")
+      expect(result.expires_on).to eq("2021-06-05")
```

Comment:
This raises a warning while running the spec suite. Should we remove it? or should we actually expect the warning message but suppress its printing?

Reply:
This is expect since we should test the method. I don't think it is a big deal. we can let the warning be printed. wdyt?

Reply:
I noticed the same and I think it's fine. The spec does help us confirming that we are correctly populating `@expires_on` and that the attribute reader is working as expected.